lib/common.sh

```diff
@@ -113,6 +113,15 @@ knownFile() {
     fi
 }
 
+getFileURL() {
```

Comment:
Maybe add a comment to explain what this function does? It's not straightforward just by reading it..

Reply:
done